mycore.org/content/de/documentation/getting_started/gs_jpa.html

```diff
@@ -2,194 +2,404 @@
+            <property name="hibernate.hikari.maximumPoolSize" value="30" />
+            <property name="hibernate.hikari.leakDetectionThreshold" value="9000" />
+            <property name="hibernate.hikari.registerMbeans" value="true" />
+
+        </properties>
+    </persistence-unit>
 </persistence>
 {{< /highlight >}}
 {{< /mcr-figure >}}
 
-      <h3>Hinweis bei Performance-Problemen</h3>
-  
-      <p>Sollt die Anwendung wiederholt abstürzen, da es mit der Anzahl der freien Datenbankverbindungen 
-      Ausnahmen gibt, könnte folgende zusätzliche Konfiguration helfen. Tote Sessions werden dann automatisch geschlossen.</p>
-  
-{{< mcr-figure label="Konfigurationsdatei" caption="<code>resources/META-INF/persistence.xml</code>">}}
+<p>
+    Außerdem muss für die Verwendung von HikariCP der Treiber in der pom.xml der Anwendung hinzugefügt werden.
+</p>
+
+{{< mcr-figure label="Konfigurationsdatei" caption="<code>pom.xml</code>">}}
 {{< highlight xml "linenos=table" >}}
-      <property name="hibernate.c3p0.unreturnedConnectionTimeout" value="1800" />
-      <property name="hibernate.c3p0.debugUnreturnedConnectionStackTraces" value="true" />
+<dependency>
+    <groupId>org.hibernate.orm</groupId>
+    <artifactId>hibernate-hikaricp</artifactId>
+    <version>${hibernate.version}</version>
+</dependency>
+<dependency>
+    <groupId>com.zaxxer</groupId>
+    <artifactId>HikariCP</artifactId>
+    <version>5.0.1</version>
+</dependency>
 {{< /highlight >}}
 {{< /mcr-figure >}}
-  
-      <h2>Konfiguration ab Release 2023.06</h2>
 
-      <p>
-        Ab Release 2023.06 wird empfohlen für das Connection-Pooling Hikari zu verwenden, da es ggf. mit C3P0 zu Problemen
-        kommen kann.
-      </p>
-      
-      <h3>pom.xml</h3>
-      
-      <p>
-        In der pom.xml der Anwendung sind nachfolgende dependencies zu ergänzen.
-        
-{{< mcr-figure label="Konfigurationsdatei" caption="<code>pom.xml</code>">}}
-{{< highlight xml "linenos=table" >}}
-    <dependency>
-      <groupId>org.hibernate.orm</groupId>
-      <artifactId>hibernate-hikaricp</artifactId>
-      <version>${hibernate.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>com.zaxxer</groupId>
-      <artifactId>HikariCP</artifactId>
-      <version>5.0.1</version>
-    </dependency>
+<h4>ab Version 2024.06</h4>
+
+<p>
+    Ab Release 2024.06 wird empfohlen die Konfiguration über die mycore.properties zu realisieren.
+    Die persistence.xml wird weiterhin unterstützt, jedoch bietet die Konfiguration per mycore.properties einige
+    Vorteile.</p>
+<ul>
+    <li>Die Mappings werden automatisch konfiguriert, je nachdem welche MyCoRe-Komponenten in der Anwendung verwendet
+        werden.
+    </li>
+    <li>Die Konfiguration ist einfacher und übersichtlicher.</li>
+    <li>Die Konfiguration kann bereits in der jar-Datei enthalten sein und Urls, Passwörter und Benutzer können in
```

Comment:
besser "URLs" (was ist der "Standard" bei uns?)

Reply:
Naja aktuell (2023.06) ist es so das alles in der persistence.xml stehen muss und ansonsten kann das jeder Anwender für sich entscheiden. Bei MIR speichern wir alles in den mycore.properties im Home-Verzeichnis. Denkbar wäre aber auch allgemein gültige Konfiguration, wie z.B. die Konfiguration des Connection-Pools, direkt im Jar zu hinterlegen.

Reply:
Achso 🤣